zerver/finders.py

```diff
@@ -11,7 +11,7 @@ class ExcludeUnminifiedMixin(object):
     def list(self, ignore_patterns):
         # We can't use ignore_patterns because the patterns are
         # applied to just the file part, not the entire path
-        excluded = '^(js|styles|templates)/'
+        excluded = '^(js|templates)/'
```

Comment:
OK, that makes sense; I guess if we wanted to preserve the behavior of not shipping unminified CSS, we could achieve that by making `tools/build-release-tarball` delete `static/styles`?